org.gnome.Snapshot.json

```diff
@@ -0,0 +1,212 @@
+			"/bin/*",
+			"/etc/*",
+			"/share/metainfo"
+		    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/ximion/appstream.git",
+                            "commit": "e4a7daefca9dcd600eea895332b8ba598cacd694"
+                        }
+                    ],
+		    "modules": [
+			{
+			    "name": "libyaml",
+			    "sources": [
+				{
+				    "type": "git",
+				    "url": "https://github.com/yaml/libyaml.git",
+				    "commit": "2c891fc7a770e8ba2fec34fc6b545c672beb37e6"
+				}
+			    ]
+			},
+			{
+                            "name": "libxmlb",
+			    "buildsystem": "meson",
+			    "config-opts": [
+				"-Dintrospection=false",
+				"-Dgtkdoc=false",
+				"-Dstemmer=false",
+				"-Dcli=false",
+				"-Dtests=false"
+			    ],
+                            "sources": [
+                                {
+                                    "type": "git",
+                                    "url": "https://github.com/hughsie/libxmlb.git",
+                                    "commit": "a6cac6a715d57c393bbddee4d0381ad943341af9"
+                                }
+                            ]
+                        }
+		    ]
+                },
+                {
+                    "name": "libsass",
+                    "buildsystem": "meson",
+                    "cleanup": [
+                        "*"
+                    ],
+		    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/lazka/libsass.git",
+                            "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
+                        }
+                    ]
+                },
+                {
+                    "name": "sassc",
+                    "buildsystem": "meson",
+		    "cleanup": [
+			"*"
+		    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/lazka/sassc.git",
+                            "commit": "82803377c33247265d779af034eceb5949e78354"
+                        }
+                    ]
+                },
+                {
+                    "name": "gtk",
+                    "buildsystem": "meson",
+                    "config-opts": [
+                        "-Dintrospection=disabled",
+                        "-Ddemos=false",
+                        "-Dbuild-testsuite=false",
+                        "-Dbuild-examples=false",
+                        "-Dbuild-tests=false"
+                    ],
+		    "cleanup": [
+			"/bin/*"
+		    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://gitlab.gnome.org/GNOME/gtk.git",
+                            "commit": "9364da673f56b61be3c76de8d79f6f9a9408409f"
```

Comment:
the version in the runtime isn't enough?  Why not specifying a tag here?

Reply:
No, we need GtkButton:can-shrink for Libadwaita. Since this was merged recently, it's not in the runtime yet

Reply:
To expand on this, it was added like 3 days ago,  and it wont make it to 44 in a point release.